### PR TITLE
RF-1.0.3.12 Support for Aries Cloud Agent Development Environment

### PR DESCRIPTION
**Pull request by asa1997**
- Comments from pull #168 has been addressed
- Name changed from Aries-dev to aries-dev
- File name for test script changed from test-kob-Aries-dev.sh to test-kobman-aries-dev.sh

### CODE REVIEW

**Comment by asa1997** [comment]
@EtricKombat  Please follow the instructions given in pull #168 for testing

**Comment by EtricKombat-zz** [comment]
@asa1997   , encountered with an error while running the test script  (failed)

image


 please refer the snap shot for the rest of the commands (success)

image

image

**Comment by EtricKombat-zz** [comment]
@asa1997  , previously reported error has been resolved with the newly commited code,


image

File: src/main/bash/commands/kobman-list.sh [hunk 1]
```diff
@@ -2,12 +2,10 @@
 
 function __kob_list {
 
-cat $KOBMAN_DIR/var/list.txt > tmp.txt
+# cat $KOBMAN_DIR/var/list.txt > tmp.txt
```
Comment (EtricKombat-zz) [inline src/main/bash/commands/kobman-list.sh]: @asa1997  , please note this for removing along with next pull requests

File: src/main/bash/commands/kobman-list.sh [hunk 2]
```diff
@@ -2,12 +2,10 @@
...
 __kobman_echo_no_colour "Available environments and their respective version numbers"
 __kobman_echo_no_colour "---------------------------------------------------------------"
-sed 's/,/ /g' tmp.txt > tmp1.txt
-sed 's/,/ /g' tmp.txt | cut -d " " -f 1 > tmp2.txt
-cat tmp1.txt | grep -h -f tmp2.txt > tmp3.txt
-cat tmp3.txt
-rm tmp*.txt
+sed 's/,/ - /1' $KOBMAN_DIR/var/list.txt
+__kobman_echo_no_colour ""
+# rm tmp*.txt
```
Comment (EtricKombat-zz) [inline src/main/bash/commands/kobman-list.sh]: @asa1997 , this as well for removal

File: src/main/bash/commands/kobman-remove.sh [hunk 3]
```diff
@@ -4,17 +4,25 @@ function __kob_remove
...
     __kobman_echo_red "This operation would remove the KOBman utility and installed environments"
     __kobman_interactive_uninstall || return 1 
-    if [[ -d $KOBMAN_ENV_ROOT ]]; then
-        __kobman_echo_no_colour "Removing dev environment"
-        rm -rf $KOBMAN_ENV_ROOT
-    fi
-
+    # if [[ -d $KOBMAN_ENV_ROOT ]]; then
+    #     __kobman_echo_no_colour "Removing dev environment"
+    #     rm -rf $KOBMAN_ENV_ROOT
```
Comment (EtricKombat-zz) [inline src/main/bash/commands/kobman-remove.sh:9]: @asa1997 , can remove if not necessary

File: src/main/bash/envs/kobman-aries-dev.sh [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+function __kobman_create_aries_dev_environment 
+{
+	
+	local environment=$1
+    local version=$2
+    __kobman_echo_white "Creating Dev environment for ${environment} under $KOBMAN_ARIES_ENV_ROOT"
+    # __kobman_echo_white "from https://github.com/${KOBMAN_USER_NAMESPACE}/${environment}"
+    __kobman_echo_white "version :${version} "
+	mkdir -p $KOBMAN_ARIES_ENV_ROOT
+	# git clone -q https://github.com/$KOBMAN_USER_NAMESPACE/${environment} $KOBMAN_ARIES_ENV_ROOT/$environment
```
Comment (EtricKombat-zz) [inline src/main/bash/envs/kobman-aries-dev.sh:34]: @asa1997 , need to be removed along with the nxt pull . ( only if not required )

**Review by EtricKombat-zz — COMMENTED** [review]
@asa1997 ,Only needs to remove unwanted commented lines , which can be done along with the next pull requests

**Comment by EtricKombat-zz** [comment]
@sumodgeorge @panickervinod , this pull request has been reviewed . 
LGTM

**Comment by asa1997** [comment]
@panickervinod  The conflicts has been resolved.

File: dist/list.txt [hunk 5]
```diff
@@ -1 +1,2 @@
-KOBman,0.0.1
```
Comment (panickervinod) [inline dist/list.txt:1]: @asa1997 shouldn't the FQRN (Fully qualified repo name)  be here ?

**Pull request merged**